### DISCUSSION TITLE
Support JSON response structure

### DISCUSSION THREAD

**Discussion by zenire** [dedoc/scramble#466] [Ideas]
Often I do not return the whole object on creation of the object via API, since the submitter already has this data. I only return properties they didn't know, like the ID:

```
return response()->json([
            'data' => [
                'id' => $user->id,
                'uuid' => $user->uuid,
            ],
        ], 201);
```

I would expect Scramble to detect the `id` and `uuid` properties and represent this response, and use it as a response example.

**Comment by romalytvynenko** [comment]
@zenire few questions.

1. Which version of Scramble do you use?

2. Please show the entire controller's method code. This code should work already, so if it doesn't, I'd like to reproduce and fix.

**Comment by zenire** [comment]
1. Version [v0.11.5](https://github.com/dedoc/scramble/releases/tag/v0.11.5)
2. Hereby a simplified version of the controller:

```
<?php

namespace App\Http\Controllers;

use App\Http\Controllers\Controller;
use App\Http\Requests\API\StoreUserRequest;
use App\Models\User;
use App\Services\UserService;
use Illuminate\Http\JsonResponse;

class UserController extends Controller
{

    public function __construct(private UserService $userService)
    {
        $this->authorizeResource(User::class, 'user');
    }


    public function store(StoreUserRequest $storeUserRequest): JsonResponse
    {
        $validated = $storeUserRequest->validated();
        $user User::create($validated);

        return response()->json([
            'data' => [
                'id' => $user->id,
                'uuid' => $user->uuid,
            ],
        ], 201);
    }

```
